changelogs/fragments/1052-aws_kms-stabilize-integration-tests.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - aws_kms - fix some bugs in integration tests and add check mode support for key rotation as well as a short wait after update (https://github.com/ansible-collections/community.aws/pull/1052).
```

Comment:
Nit, could you add some words explaining why we wait after update?  Something like, "for requested changes to be reflected on AWS" maybe?  Otherwise folks might be confused why the module takes longer to run updates now.